pkg/rancher-desktop/pages/snapshots/create.vue

```diff
@@ -0,0 +1,158 @@
+<script lang="ts">
+
+import { Banner, LabeledInput, TextAreaAutoGrow } from '@rancher/components';
+import dayjs from 'dayjs';
+import Vue from 'vue';
+
+const defaultName = () => {
+  const dateString = dayjs().format('YYYY-MM-DD_HH_mm_ss');
+
+  return `snap_${ dateString }`;
+};
+
+interface Data {
+  name: string,
+  notes: string,
+  creating: boolean,
+}
+
+interface Methods {
+  goBack: () => void;
+  submit: () => void;
+}
+
+interface Computed {
+  valid: boolean;
+}
+
+export default Vue.extend<Data, Methods, Computed, never>({
+  components: {
+    Banner,
+    LabeledInput,
+    TextAreaAutoGrow,
+  },
+
+  data() {
+    return {
+      name:     defaultName(),
+      notes:    '',
+      creating: false,
+    };
+  },
+
+  computed: {
+    valid() {
+      return !!this.name;
+    },
+  },
+
+  mounted() {
+    this.$store.dispatch(
+      'page/setHeader',
+      { title: this.t('snapshots.create.title') },
+    );
+    (this.$refs.nameInput as any)?.focus();
```

Comment:
Is the `v-focus` directive not functioning properly?

Reply:
This can be removed.